content/cpp/concepts/data-types/data-types.md

```diff
@@ -94,22 +108,27 @@ double weight1;
 int weight2;
 
 weight1 = 154.49;
+// assigned a double value
 weight2 = (int) weight1;
-
-// weight2 is now 154
+// weight2 is now 154 due to explicit type conversion
+int weight3 = weight1;
+// weight3 is also 154 due to implicit conversion by the compiler
 ```
 
 **Note:** Going from a double to an int simply removes the decimal. There’s no rounding involved.
 
-## Memory Size
+Alternatively, there is a safer version of casting in C++ called `static_cast` that can be used. Performed at compile time, `static_cast` offers better debugging and safer code:
 
-As noted above, the data type of a variable also specifies the size of the information that it can hold:
+```cpp
+double weight1 = 122.03;
+int weight2 = static_cast<int>(weight1);
+std::cout << weight2 << std::endl;
+// 122
+```
 
-| Data Type     | Memory Size |
-| ------------- | ----------- |
-| `bool`        | 1 byte      |
-| `char`        | 1 byte      |
-| `int`         | 4 bytes     |
-| `float`       | 4 bytes     |
-| `double`      | 8 bytes     |
-| `std::string` | 24 bytes    |
+**Note:** Not all types can be converted:
+```cpp
+std::string s = static_cast<std::string>(weight2);
+```
+
+> note: no known conversion for argument 1 from ‘int’ to ‘std::__cxx11::basic_string<char>&&’
```

Comment:
Is this supposed to be a `> note:` or an error message? If the latter, can we made an edit like this:

```suggestion
This throws the following error:

```

And then attach a \```shell snippet with the error:

```shell
no known conversion for argument 1 from ‘int’ to ‘std::__cxx11::basic_string<char>&&’
````